pkg/log/log.go

```diff
@@ -121,6 +125,17 @@ func SetFormatter(format OutputFormat) {
 func init() {
 	SetFormatter(TextFormat)
 	initEnvVarFields()
+	level, err := logrus.ParseLevel(os.Getenv(LevelVarName))
+
+	if err != nil {
+		level = logrus.InfoLevel
+	}
```

Comment:
Should we move this under `SetLevel()`?

Reply:
@PrasadG193 What if we want to set it explictly in code, using SetLevel() rather than using the env variable?